yarn.lock

```diff
@@ -3579,7 +3579,6 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "@radix-ui/react-polymorphic@workspace:packages/react/polymorphic"
   dependencies:
-    "@babel/runtime": ^7.13.10
```

Comment:
Looks like we forgot to `yarn` in this PR https://github.com/radix-ui/primitives/pull/670